docs/user-documentation/intro-to-claw.md

```diff
@@ -62,6 +62,42 @@ Islandora CLAW is developed _by_ the Islandora community, _for_ the Islandora co
 
 ### Linked Data
 
+Linked Data is a big concept. A [definition by Tim Bernes-Lee](https://www.w3.org/DesignIssues/LinkedData.html) from 2006 articulates the basic idea in the context of the "Semantic Web":   
+
+>It is about making links, so that a person or machine can explore the web of data.  With linked data, when you have some of it, you can find other, related, data.
+
+>Like the web of hypertext, the web of data is constructed with documents on the web. However,  unlike the web of hypertext,  where links are relationships anchors in hypertext documents written in HTML, for data they links  between arbitrary things described by RDF,.  The URIs identify any kind of object or  concept.   But for HTML or RDF, the same expectations apply to make the web grow:
+
+> 1. Use URIs as names for things
+
+> 1. Use HTTP URIs so that people can look up those names.
+
+> 1. When someone looks up a URI, provide useful information, using the standards (RDF*, SPARQL)
+
+> 1. Include links to other URIs. so that they can discover more things.
+
+LDP is an official W3C recommendation. According to [their spec](https://www.w3.org/TR/ldp/), Linked Data Platform (LDP) defines a set of rules for HTTP operations on web resources, some based on RDF, to provide an architecture for read-write Linked Data on the web. Put more simply: An LDP server is a web server that lets you manage relationships in RDF by creating URLs you can perform CRUD (Create Read Update Delete)
+operation on:
+* `http://my_url/some_resource`
+* `http://my_url/some_resource/children`
+* `http://my_url/some_resource/children/child_resource`
+* `<> prefix:isChildOf <http://my_url/some_resource>`
+
+An LDP server also lets you request the RDF graph for a resource with varying levels of additional information for discovery:
+* The resource itself
+* The resource plus child resources
+* The resource plus all other resources that reference it (inbound references)
+* The resource plus children plus inbound references
+
+Fedora 4 is an LDP implementation. It stores metadata using RDF (Resource Description Framework), a standard format for data interchange on the web. RDF data is formatted as triples, like so: `<resource_uri> <predicate_uri> <object_uri>` and can be exported in many different formats:
+
+* JSON-LD
+* RDF/XML
+* Turtle
+* N-Triples
+
+Working with RDF is not entirely new in the Islandora/Fedora context, as RELS-EXT and RELS-INT have always been RDF/XML. Related objects can be included in RDF, so it functions as a graph you can traverse in order to discover more information.
+
```

Comment:
This is good 👍 

You could add an example using Dublin Core as XML and RDF using the dc namespace.

ie. 
Dublin Core XML you store with the resource.
```
<?xml version="1.0"?>
<metadata
  xmlns="http://example.org/myapp/"
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xsi:schemaLocation="http://example.org/myapp/ http://example.org/myapp/schema.xsd"
  xmlns:dc="http://purl.org/dc/elements/1.1/">
  <dc:title>
    Islandora
  </dc:title>
  <dc:description>
    Islandora is an open-source software framework designed to help institutions and organizations and their audiences collaboratively manage, and discover digital assets using a best-practices framework.
  </dc:description>
</metadata>
```
vs RDF (which is actually properties of the resource)
```
@prefix dc: <http://purl.org/dc/elements/1.1/> .
<http://islandora.ca/> dc:title "Islandora" ;
   dc:description "Islandora is an open-source software framework designed to help institutions and organizations and their audiences collaboratively manage, and discover digital assets using a best-practices framework." .
```

This is not a great example as there is the issue of http://islandora.ca being a webpage for the project of Islandora. But I don't care.